### PR TITLE
Round entries of vertices.

### PR DESCRIPTION
**Pull request by dhendryc**
Round the entries of vertices corresponding to the integer variables. 

To clean up and avoid weird behavior.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -9,6 +9,7 @@ DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 FrankWolfe = "f55ce6ea-fdc5-4628-88c5-0087fe54bd30"
+HiGHS = "87dc4568-4c63-4d18-b0c0-bb2238e4078b"
```
Comment (matbesancon) [inline Project.toml:12]: not as a main dependency normally

File: src/time_tracking_lmo.jl [hunk 2]
```diff
@@ -29,6 +33,12 @@ function FrankWolfe.compute_extreme_point(lmo::TimeTrackingLMO, d; kwargs...)
     cleanup_solver(lmo.lmo.o)
     v = FrankWolfe.compute_extreme_point(lmo.lmo, d; kwargs)
 
+    if !is_linear_feasible(lmo, v)
+        @debug "Vertex not linear feasible $(v)"
+        @assert !is_linear_feasible(lmo, v)
```
Comment (matbesancon) [inline src/time_tracking_lmo.jl:38]:
```suggestion
        @assert is_linear_feasible(lmo, v)
```

Comment (matbesancon) [reply src/time_tracking_lmo.jl:38]: otherwise the assert is never triggered because we are in the condition

File: Project.toml (hunk 1 above)
Comment (dhendryc) [reply Project.toml:12]: Don't we need HiGHS for the strong branching?

Comment (matbesancon) [reply Project.toml:12]: not in the package itself, only in the tests.

Comment (dhendryc) [reply Project.toml]: okay, no, I just checked. The user chooses the optimizer for strong branching.

**Pull request merged**